voluncheer/opportunityboard/views/search.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from django.shortcuts import render
+
+from opportunityboard.models import Category
+from opportunityboard.models import Opportunity
+from opportunityboard.models import Subcategory
+from opportunityboard.models import Subsubcategory
+from opportunityboard.views.opportunityboard import category_dict_gen
+
+
+def filter_search(request):
+    if request.method == "POST":
+        filter = parse_search_filter(request.POST)
```

Comment:
It's not clear to me what the `if request.method == "POST"` branch is trying to accomplish. Can you provide a docstring or some kind of indication as to why we are logging an error when the request is a post?

Reply:
this will be useful as protection after introducing filter search functions. For now it's just a log dump